src/com/nekeki/kawaiiparkour/TimeManager.java

```diff
@@ -472,34 +475,33 @@ public static String getSetupName(Player player) {
         //Check if given location is already a point. Optional to only check if course is enabled.
         Set<String> parkourKeys = kawaiiFile.getConfigurationSection("parkour").getKeys(false);
         String[] locationSet = {"null", "null", "null"};
-        for(String x : parkourKeys) {
-            if(isEnabled && !kawaiiFile.getBoolean("parkour." + x + ".enabled")) {
-                //skip
```

Comment:
I recommend looking up 'continue' as loop function ;D
There are a lot of sites that explain it, e.g. https://www.w3schools.com/java/java_break.asp